markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -490,11 +429,6 @@ library PerpsAccount {
         // in market units
         amountToLiquidate = MathUtil.min(maxLiquidatableAmount, MathUtil.abs(position.size));
 
-        liquidationReward = MathUtil.min(
-            marketConfig.maxLiquidationReward,
-            amountToLiquidate.mulDecimal(marketConfig.liquidationRewardPercentage)
-        );
```

Comment:
i'll come back to this; we will need to figure out the granular details of how liquidation rewards will work next.